tests/EventTest.php

```diff
@@ -91,6 +92,7 @@ public function testEventIsGeneratedWithUniqueIdentifier(): void
     public function testToArray(): void
     {
         $this->options->setRelease('1.2.3-dev');
```

Comment:
Is this line really needed? I can't see how this affect the the last assertion since we're never checking the `release` attribute of the returned payload